modules/apps/asset/asset-auto-tagger-google-cloud-natural-language-impl/src/main/java/com/liferay/asset/auto/tagger/google/cloud/natural/language/internal/GCloudNaturalLanguageDocumentAssetAutoTagProvider.java

```diff
@@ -66,12 +85,147 @@ public Collection<String> getTagNames(AssetEntry assetEntry) {
 		return Collections.emptyList();
 	}
 
+	public Collection<String> getTagNames(
```

Comment:
Hey @regiCesar21, can we make these moved methods private and remove the not used if there are? Thx